spec/lib/jobs/install-os-job-spec.js

```diff
@@ -379,5 +379,28 @@ describe('Install OS Job', function () {
                    .to.throw('The uid should between 500 and 65535 (>=500, <=65535)');
         });
         
+        it('should throw error when mountPoint is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:{}, size:'500', fsType:'ext3' }];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('mountPoint must be a string');
+        });
+
+        it('should throw error when size is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:'/boot', size:{}, fsType:'ext3' }];
```

Comment:
I would expect to test more invalid types, not just `{}` as user may input any kind of types. So we need array:
```javascript
[{}, 'foo', [1], null, undefined, NaN, 123].forEach(function(value) {
    job.options.installPartitions = [{ mountPoint:'/boot', size: value, fsType:'ext3' }];
    return expect(job._validateOptions.bind(job,{})).to.throw('size must be a string');
});
```
The same for other test cases.